lib/manageiq/providers/ibm_cloud/cloud_tools/sdks/vpc.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+require 'ibm_vpc'
+
+dir_name = File.basename(__FILE__).split('.')[0]
+Dir.glob(File.join(dir_name, '*.rb'), :base => File.dirname(__FILE__)) { |req| require_relative req }
+
+module ManageIQ
+  module Providers
+    module IbmCloud
+      module CloudSDK
```

Comment:
The module path should match the file path for consistency (and for when we get to Zeitwerk)

```
"ManageIQ::Providers::IbmCloud::CloudSDK::Vpc".underscore + ".rb"
# => "manageiq/providers/ibm_cloud/cloud_sdk/vpc.rb"
```

Reply:
Ok, I've reworked the namespacing to follow the directory structure.